### PR TITLE
fix(wallet) Update backup wallet page to onboarding v3

### PR DESCRIPTION
**Pull request by muliswilliam**
Resolves https://github.com/brave/brave-browser/issues/38640

## Submitter Checklist:

- [x] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:

https://github.com/brave/brave-core/assets/48117473/e5417524-f886-466e-a4b2-72ec8d32e0eb

### CODE REVIEW

File: components/brave_wallet_ui/page/screens/backup-wallet/backup-enter-password/backup-enter-password.tsx [hunk 1]
```diff
@@ -97,54 +93,46 @@ export const BackupEnterPassword: React.FC = () => {
...
-              text={getLocale('braveWalletButtonContinue')}
-              disabled={!password || !isCorrectPassword}
-              onSubmit={onSubmit}
-            />
-          </NextButtonRow>
-        </StyledWrapper>
-      </MainWrapper>
-    </CenteredPageLayout>
+    <OnboardingContentLayout
+      showBackButton
```
Comment (Douglashdaniel) [inline components/brave_wallet_ui/page/screens/backup-wallet/backup-enter-password/backup-enter-password.tsx:99]: What does the `back` button do in this instance?

**Comment by josheleonard** [comment]
Screenshot 2024-05-28 at 10 48 08 AM

Can we remove the border on the "eye" icon?

**Comment by muliswilliam** [comment]
> Screenshot 2024-05-28 at 10 48 08 AM
> Can we remove the border on the "eye" icon?

That is how the design was.

File: components/brave_wallet_ui/page/screens/backup-wallet/backup-enter-password/backup-enter-password.tsx (hunk 1 above)
Comment (muliswilliam) [reply components/brave_wallet_ui/page/screens/backup-wallet/backup-enter-password/backup-enter-password.tsx:99]: Navigates the user back to portfolio

**Review by Douglashdaniel — APPROVED** [review]
++

**Comment by srirambv** [comment]
Verification passed on  

Brave | 1.68.64 Chromium: 126.0.6478.26 (Official Build) nightly (64-bit)
-- | --
Revision | d9689f56dafd3c9d8622461f6b4864053b0706aa
OS | Linux

- Verified steps from PR description
- Verified Backup wallet from portfolio loads the new UI 

https://github.com/brave/brave-core/assets/17010094/572f0801-fa88-4f84-8956-053efa1efb4d

**Pull request merged**